models/UserManager.class.php

```diff
@@ -3,13 +3,45 @@
 class UserManager {
 	private $_user;
 
+    public function checkMail($mail) {
+        if (strlen($mail) > 30) {
```

Comment:
I think it is possible to have a mail longer than 30 char

Reply:
lol non